### PR TITLE
Fixed to AS2 Connector Versions

### PR DESCRIPTION
**Pull request by dhirsch1**
Fixed the landing page for AS2 and also fixed the Mule 3 topic titles.

### CODE REVIEW

**Review by mcherem — COMMENTED** [review]
This is a Mule 4 doc and should be removed from here.

File: as2/3.0/modules/ROOT/pages/as2-connector-examples.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= AS2 - Examples - Mule 4
+= AS2 - Examples - Mule 3
```
Comment (mcherem) [inline as2/3.0/modules/ROOT/pages/as2-connector-examples.adoc]: This is a Mule 4 example and should be removed from here.

File: as2/3.0/modules/ROOT/pages/as2-connector-studio.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= AS2 - Studio Configuration - Mule 4
+= AS2 - Studio Configuration - Mule 3
```
Comment (mcherem) [inline as2/3.0/modules/ROOT/pages/as2-connector-studio.adoc]: Again, Mule 4 example. Please remove.

File: as2/3.0/modules/ROOT/pages/as2-connector-xml-maven.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-= AS2 - XML and Maven Support - Mule 4
+= AS2 - XML and Maven Support - Mule 3
```
Comment (mcherem) [inline as2/3.0/modules/ROOT/pages/as2-connector-xml-maven.adoc]: Mule 4 example, please remove.

**Review by mcherem — COMMENTED** [review]
Added comments inline about which files should be removed.

**Review by valkyrie69 — APPROVED** [review]
Assuming you ran the link checker.

**Pull request merged**